drivers/led/led_shell.c

```diff
@@ -353,6 +394,10 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 		      3, 0),
 	SHELL_CMD_ARG(on, &dsub_device_name, SHELL_HELP("Turn on LED", "<device> <led>"), cmd_on, 3,
 		      0),
+	SHELL_CMD_ARG(blink, &dsub_device_name,
+		      SHELL_HELP("Blink LED on and off",
+				 "<device> <led> <delay_on_in_ms> [delay_off_in_ms]"),
```

Comment:
Thought I had suggested it this way, apologies if not! Not blocking, I guess

```suggestion
				 "<device> <led> <delay_on_in_ms> [<delay_off_in_ms>]"),
```

Reply:
You're absolutely right. Done. BTW, just so you know, there are a bunch of places where it's written `[param]` but I agree with you that it should be like this